OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -28,6 +28,9 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Maximum offset at the maximum range.")]
```

Comment:
I'd wait with that before someone decides if @UberWaffe's suggestion should be incorporated into this one or not.